### DISCUSSION TITLE
Quantos acessos usando o certificado é permitido por dia?

### DISCUSSION THREAD

**Discussion by Iazzetta** [andreroggeri/pynubank#366] [Q&A] [answered]
Tenho um projeto rodando que acessa de 5 em 5 segundos o projeto. Hoje de manhã vi um erro de `Max retries exceeded` ao acessar o  `/api/query` e não está mais conectando na conta e buscando extrato. Como posso contornar isso para não me preocupar com isso?

**Comment by andreroggeri** [comment, marked as answer]
Não tem isso documentado, mas tem [essa issue](https://github.com/andreroggeri/pynubank/issues/238) onde alguns usuários discutiram isso.

**Comment by jeffbzrr** [comment]
Eu acesso a cada hora. Sem nenhum problema. Já vi pessoal com intervalo menor.

**Comment by Iazzetta** [comment]
Consegui melhorar com um banco de dados. Eu gero o pedido pelo sistema e só com pedidos pendentes eu inicializo o pynubank e pego 50 registros para validar. Dessa forma ele não fica acessando sempre a conta, só quando necessário.
